.github/workflows/lint_commit_messages.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint Commit Messages
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Setup gitlint
+        run: sudo apt-get install gitlint
+      - name: Checkout
+        run: git clone https://github.com/SUSE/carrier.git && cd carrier && git checkout ${GITHUB_HEAD_REF}
+      - name: Lint new commits
+        working-directory: ./carrier
+        run: gitlint --commits "origin..HEAD" --contrib contrib-title-conventional-commits  --ignore B6
```

Comment:
It seems we can have a `.gitlint` file for configuration? https://jorisroovers.com/gitlint/#configuration

Reply:
True, that might be better, especially for the git hook. I will check if contrib extensions are also supported like this.